.github/scripts/build-macOS.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -e
+
+if [[ "${GITHUB_REF_TYPE}" == "tag" ]]; then
+    BUILD_TAG=${GITHUB_REF_NAME}
+else 
+    BUILD_TAG=$(git rev-parse --short HEAD)
+fi
+
+# install deps
+brew install SDL2
+
+# build
+cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo -DBUILD_TESTS=ON -DDETHRACE_WERROR=ON -B build
+cmake --build build -- -j 4
+
+# package artifact
+tar -czvf dethrace-${BUILD_TAG}-darwin-amd64.tar.gz build/dethrace
+
+echo "::set-output name=filename::dethrace-${BUILD_TAG}-darwin-amd64.tar.gz"
```

Comment:
This is just cosmetic, but can you please add a final new line to all files to avoid GitHub adding this marker.

Reply:
thanks. I moved to a new laptop and forgot to turn on the auto-insert newlines setting 👍

Reply:
On my dev machine, I added a `.editorconfig` in the root folder of my user home dir containing:
```
root = true

[*]
end_of_line = lf
insert_final_newline = true

[Makefile]
indent_style = tab
indent_size = 4

[*.py]
indent_style = space
tab_width = 4
```
And all my ide's are configured to use editorconfig files.

When some project has another default, I add an `.editorconfig` file in that projects root directory.
To avoid git picking up that file, I added `.editorconfig` to `.gitignore` in my user home directory.